### DISCUSSION TITLE
Tasklist Zim 0.75 bugs ?

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#2173] [General]
1) Task list settings are not saved after the list window is closed.

2) Dates of closed tasks remain red.

Xubuntu 22.04

**Comment by reagle** [comment]
Confirmed #1. If I set to show to "Active" only, and then terminate and rerun, the setting has been lost. This should be moved to issues. (I don't understand/use point 2?)

**Comment by Nick222** [comment]
> I don't understand/use point 2

When a task is marked as completed, its date in the date column of the task list remains red.
This is illogical - the task has already been completed, why such an attention-grabbing color?

**Reply by reagle** [reply]
I wonder if this was addressed in 0.75.1 . If not, perhaps this should become an issue?

**Comment by liuxiaojin93** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
